python/sqlflow_submitter/explainer.py

```diff
@@ -30,13 +39,15 @@ def plot_and_save(plotfunc, filename='summary'):
         None
     '''
 
-    # The default backend
     plotfunc()
     plt.savefig(filename, bbox_inches='tight')
-
-    # The plotille text backend
-    matplotlib.use('module://plotille_text_backend')
-    import matplotlib.pyplot as plt_text_backend
-    sys.stdout.isatty = lambda: True
-    plotfunc()
-    plt_text_backend.savefig(filename, bbox_inches='tight')
+    if is_pai:
+        utils.copyfileobj(filename + '.png', oss_dest, oss_ak, oss_sk,
+                          oss_endpoint, oss_bucket_name)
+    else:
```

Comment:
Remove the `else:`? We may support using a terminal to submit pai tasks in the near future.

Reply:
In fact, I tested the code in `else` block and it failed in the PAI platform, so I move such code under `else`. 
Let me add a `Note` here for the future.

Reply:
I think the reason is `plotille` is not in requirements.txt, I'll fix it later.